src/cobra/core/gene.py

```diff
@@ -299,3 +294,289 @@ def _repr_html_(self):
             n_reactions=len(self.reactions),
             reactions=format_long_string(", ".join(r.id for r in self.reactions), 200),
         )
+
+
+class GPR(Module):
```

Comment:
I think we should consider putting that into its own file like `src/cobra/core/gpr.py` there are more functions in modify.py that could move there.